sa/sa.go

```diff
@@ -453,47 +453,72 @@ func ReverseName(domain string) string {
 	return strings.Join(labels, ".")
 }
 
+// TODO(@cpu): This query can be removed when UseRenewalFirstRL is the default.
```

Comment:
This comment and the one below reference `UseRenewalFirstRL` but I don't see that defined anywhere. I'm guessing you meant `Allow`?

Reply:
Good catch, you're correct. Fixed in fb2f67f